meta/types/bucket.go

```diff
@@ -11,6 +11,12 @@ import (
 	"time"
 )
 
+const(
+	VersionEnabled   = "Enabled"
+	VersionDisabled  = "Disabled"
+	VersionSuspended ="Suspended"
```

Comment:
Leave a blank space before "Suspended",maybe u should use gofmt to format your code before commit.

Reply:
Fixd it